src/app/lib/utilities/preprocessor/rules/cpsAssetPage/addHeadlineBlock/index.js

```diff
@@ -1,57 +1,57 @@
 import pathOr from 'ramda/src/pathOr';
+import deepClone from 'ramda/src/clone';
+import path from 'ramda/src/path';
+import {
+  getOffScreenHeadlineBlock,
+  getOnScreenHeadlineBlock,
+  getHeadlineBlock,
+} from './models';
 
-const addHeadingToSTY = jsonRaw => {
-  const headlineText = pathOr(
-    null,
-    ['promo', 'headlines', 'headline'],
-    jsonRaw,
-  );
+const getHeadlineBlocks = json => {
+  const headlineText = pathOr(null, ['promo', 'headlines', 'headline'], json);
 
   if (!headlineText) {
-    return jsonRaw;
+    return json;
   }
 
-  const newBlock = {
-    model: {
-      blocks: [
-        {
-          model: {
-            blocks: [
-              {
-                model: {
-                  blocks: [
-                    {
-                      model: {
-                        attributes: [],
-                        text: headlineText,
-                      },
-                      type: 'fragment',
-                    },
-                  ],
-                  text: headlineText,
-                },
-                type: 'paragraph',
-              },
-            ],
-          },
-          type: 'text',
-        },
-      ],
-    },
-    type: 'headline',
-  };
-
   return {
-    ...jsonRaw,
-    content: {
-      model: {
-        blocks: [
-          newBlock,
-          ...pathOr([], ['content', 'model', 'blocks'], jsonRaw),
-        ],
-      },
-    },
+    offScreenHeadlineBlock: getOffScreenHeadlineBlock(headlineText),
+    onScreenHeadlineBlock: getOnScreenHeadlineBlock(headlineText),
+    headlineBlock: getHeadlineBlock(headlineText),
   };
 };
 
-export default addHeadingToSTY;
+const firstBlockIsVideo = blocks => path(['0', 'type'], blocks) === 'video';
```

Comment:
Just for some compatibility with the live stream PR:

```suggestion
const firstBlockIsVideo = blocks => ['video', 'version'].includes(path(['0', 'type'], blocks));
```

Reply:
Are we converting the live stream into a `version` block? I assumed it would be converted into a `video` block. EG: `version` would be considered a child block of `video`

Reply:
What we get from ares is already a "version" block - I just wasn't planning on changing it as it can't be made identical to a "true" `video` block schema

Can go either way with it though - not a big deal